packages/elastic-sip-trunking/README.md

```diff
@@ -0,0 +1,113 @@
+
+However, it's still possible to use this SDK standalone is you need to access the Elastic SIP Trunking API only.
+
+### With NPM
+
+```bash
+npm install @sinch/elastic-sip-trunking
+```
+
+### With Yarn
+
+```bash
+yarn add @sinch/elastic-sip-trunking
+```
+
+## Usage
+
+### Credentials
+
+The `Elastic SIP Trunking` API uses the Sinch unified authentication with OAuth2. You will need to provide the following credentials:
+- projectId: can be found in the [Account Dashboard](https://dashboard.sinch.com/settings/access-keys)
+- keyId:: can be found in your Access key list in the [Account Dashboard](https://dashboard.sinch.com/settings/access-keys)
+- keySecret: can be found **ONLY** when generating a new access key: keep it safe!
+
+### As part of the Sinch SDK
+
+If you are using this SDK as part of the Sinch SDK (`@sinch/sdk-core`) you can access it as the `elasticSipTrunking` property of the client that you would have instantiated.
+
+```typescript
+import {
+  ElasticSipTrunkingService,
+  ElasticSipTrunking,
+  SinchClient,
+  UnifiedCredentials,
+} from '@sinch/sdk-core';
+
+const credentials: UnifiedCredentials = {
+  projectId: 'PROJECT_ID',
+  keyId: 'KEY_ID',
+  keySecret: 'KEY_SECRET',
+};
+
+// Access the 'fax' service registered on the Sinch Client
+const sinch = new SinchClient(credentials);
+const elasticSipTrunkingService: ElasticSipTrunkingService = sinch.elasticSipTrunking;
+
+// Build the request data
+const requestData: ElasticSipTrunking.InterfaceTBD = {
+};
+
+// Access the 'elasticSipTrunking' service registered on the Sinch Client
+const result = await sinch.elasticSipTrunking.tag.method(requestData);
+```
+
+### Standalone
+
+The SDK can be used standalone if you need to use only the Elastic SIP Trunking APIs.
+
+```typescript
+import {
+  UnifiedCredentials,
+} from '@sinch/sdk-client';
+import {
+  ElasticSipTrunkingService,
+  ElasticSipTrunking,
+} from '@sinch/elastic-sip-trunking';
+
+const credentials: UnifiedCredentials = {
+  projectId: 'PROJECT_ID',
+  keyId: 'KEY_ID',
+  keySecret: 'KEY_SECRET',
+};
+
+// Declare the 'elasticSipTrunking' service in a standalone way
+const elasticSipTrunkingService = new ElasticSipTrunkingService(options);
+
+// Build the request data
+const requestData: ElasticSipTrunking.InterfaceTBD = {
+};
+
+// Use the standalone declaration of the 'elasticSipTrunking' service
+const result = await elasticSipTrunkingService.tag.method(requestData);
+```
+
+## Promises
+
+All the methods that interact with the Sinch APIs use Promises. You can use `await` in an `async` method to wait for the response or you can resolve them yourself with `then()` / `catch()`.
+
+```typescript
+// Method 1: Wait for the Promise to complete
+let result: ElasticSipTrunking.ResultInterfaceTBD;
+try {
+  result = await elasticSipTrunkingService.tag.method(requestData);
+  console.log(``);
```

Comment:
This will be updated later, with some real code example. As you can see above, there are placeholders for everything: `tag.method` and `ResultInterfaceTBD`